Tests/Marketplace/zip_packs.py

```diff
@@ -244,12 +288,20 @@ def main():
     zipped_packs = []
     success = True
     try:
-        zipped_packs = download_packs_from_gcp(storage_bucket, gcp_path, zip_path, circle_build, branch_name)
+        download_packs_from_gcp(storage_bucket_name, gcp_path, zip_path, circle_build, branch_name)
     except Exception:
         logging.exception('Failed downloading packs')
         success = False
```

Comment:
Does it make sense to go on with the logic when you were not able to download from GCP?

Reply:
It seems unnecessary to me as well. It was added by @liorblob.
Is there a reason you've added it and didn't just end the entire process?

Reply:
I guess it's for the `cleanup()` at the end